src/components/interactive-builder/edit-question.modal.tsx

```diff
@@ -578,6 +621,7 @@ const EditQuestionModal: React.FC<EditQuestionModalProps> = ({
             )}
 
             {fieldType !== 'ui-select-extended' &&
+              questionToEdit.type !== 'encounterDatetime' &&
```

Comment:
This check should suffice instead of explicitly checking the rendering types (`date` and `datetime`), right, @samuelmale?

Reply:
That makes sense.